t/data/WebAPIPlugin/Hello.pm

```diff
@@ -0,0 +1,14 @@
+package WebAPIPlugin::Hello;
+use Mojo::Base 'Mojolicious::Plugin';
+use Mojo::Base 'Mojolicious::Controller';
```

Comment:
Don't do multiple inheritance, it's not worth it. Make a separate controller.

Reply:
To be more clear, this is unsupported by upstream and i will probably break that with a future Mojolicious release. 😜